src/sentry_logger.cpp

```diff
@@ -118,41 +116,28 @@ void SentryLogger::_log_error(const char *p_func, const char *p_file, int p_line
 
 	if (as_breadcrumb) {
 		// Log error as breadcrumb.
-		// TODO: Move this code to abstraction layer later when its available.
-		sentry_value_t crumb = sentry_value_new_breadcrumb("error", p_rationale);
-		sentry_value_set_by_key(crumb, "category", sentry_value_new_string("error"));
-		sentry_value_set_by_key(crumb, "level",
-				sentry_value_new_string(
-						p_error_type == ERROR_TYPE_WARNING ? "warning" : "error"));
-
-		sentry_value_t data = sentry_value_new_object();
-		sentry_value_set_by_key(data, "function", sentry_value_new_string(p_func));
-		sentry_value_set_by_key(data, "file", sentry_value_new_string(p_file));
-		sentry_value_set_by_key(data, "line", sentry_value_new_int32(p_line));
-
-		const char *error_string = error_types[p_error_type];
-		sentry_value_set_by_key(data, "godot_error_type", sentry_value_new_string(error_string));
-
-		sentry_value_set_by_key(crumb, "data", data);
-		sentry_add_breadcrumb(crumb);
+		Dictionary data;
+		data["function"] = String(p_func);
+		data["file"] = String(p_file);
+		data["line"] = p_line;
+		data["godot_error_type"] = String(error_types[p_error_type]);
```

Comment:
where are these error types maintained?

I'm thinking we could load them in the UI to give the user more context